Javascript.md

```diff
@@ -1,5 +0,0 @@
-# JavaScript: Basics and Tricky Questions
-
-### What are the differences between null and undefined?
-- `null`: No value. Null can be assigned to any variable but is not an object. typeof null returns `object`
-- `undefined`
```

Comment:
it might be a good idea to separate them, otherwise the frontend file might get too long.